config/routes.js

```diff
@@ -50,7 +49,7 @@ module.exports.routes = {
     controller: 'AuthController',
     action: 'callback'
   },
-  'get /api/data/:indicator': {
+  'get /api/data/indicator/:indicator': {
```

Comment:
@berkantesk @ogzhnozlci  must know about this. Endpoint URL changed to 
 /api/data/indicator/:indicator